src/anemoi/training/config/debug.yaml

```diff
@@ -18,10 +18,13 @@ defaults:
 
 diagnostics:
   plot:
-    enabled: False
-hardware:
-  files:
-    graph: ???
+    enabled: True
+  log:
+    mlflow:
+      enabled: True
+      authentication: True
+      tracking_uri: https://mlflow.ecmwf.int
+
 model:
```

Comment:
This should be left as it was, we should not use ecmwf things by default

Reply:
oh sorry I will change this